picoquic/quicctx.c

```diff
@@ -150,11 +150,7 @@ picoquic_quic_t * picoquic_create(uint32_t nb_connections,
 			quic->flags |= picoquic_context_unconditional_cnx_id;
 		}
 
-        if (cert_file_name != NULL)
-        {
-            quic->flags |= picoquic_context_server;
-        }
-        else if (ticket_file_name != NULL)
+        if (ticket_file_name != NULL && cert_file_name == NULL)
         {
```

Comment:
Not sure that this is the right test. The beauty of this whole PR is that it allows a single context to be used for accepting connections as server and establishing connections as client. If the context is created with a non NULL cert file name, that implies it will be able to be used as server. But if it could also be used as client, then it will need a ticket store in order to resume connections and use 0-RTT. Isn't that what the programmer indicates by setting the ticket_file_name to a non NULL value?

Reply:
Yeah I was not quite sure on what to test here.